media_softlet/linux/common/os/osservice/mos_utilities_specific.cpp

```diff
@@ -1003,7 +1003,25 @@ MOS_STATUS MosUtilitiesSpecificNext::UserFeatureDumpDataToFile(const char *szFil
     MOS_PUF_KEYLIST   pKeyTmp;
     int32_t           j;
 
+   // For release version: writes to the file if it exists,
+// skips if it does not exist
+// For other version: always writes to the file
+
+#if(_RELEASE)
```

Comment:
I suppose we dont need this macro, should not create it even with debug version. could you help to check why debug version need create it?

Reply:
Hi Xinfeng, debug/releaseinternal version need this regkey files to read/write regkey, it will be more convenient. Otherwise, we have to create it manually, it will be troublesome